include/openthread/platform/ble.h

```diff
@@ -0,0 +1,1013 @@
+/*
+ *  Copyright (c) 2018, The OpenThread Authors.
+ *  All rights reserved.
+ *
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions are met:
+ *  1. Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *  2. Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in the
+ *     documentation and/or other materials provided with the distribution.
+ *  3. Neither the name of the copyright holder nor the
+ *     names of its contributors may be used to endorse or promote products
+ *     derived from this software without specific prior written permission.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ *  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ *  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ *  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ *  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ *  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ *  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ *  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ *  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ *  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *  POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/**
+ * @file
+ *   This file defines a generic OpenThread BLE driver HOST interface.
+ */
+
+#ifndef _PLATFORM_BLE_H_
+#define _PLATFORM_BLE_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+
+#include <openthread/types.h>
+
+/**
+ * @defgroup ble BLE Host
+ * @ingroup platform
+ *
+ * @brief
+ *   This module includes the platform abstraction for BLE Host communication.
+ *
+ * @{
+ *
+ */
+
+enum
+{
+    /**
+     * The size of the Bluetooth Device Address.
+     */
+    OT_BLE_ADDRESS_LENGTH = 6,
+
+    /**
+     * Time slot duration on PHY layer in microseconds (0.625ms).
+     */
+    OT_BLE_TIMESLOT_UNIT = 625,
+
+    /**
+     * Minimum allowed connection interval in OT_BLE_TIMESLOT_UNIT units (7.5ms).
+     */
+    OT_BLE_CONN_INTERVAL_MIN = 0x0006,
```

Comment:
It looks like interval is calculated as 0x6 * 0.625 ms = 3.75ms, whereas the comment says 2x that. You probably refer to `OT_BLE_CONN_INTERVAL_UNIT`